CMakeLists.txt

```diff
@@ -67,6 +67,9 @@ add_compile_definitions(NPY_NO_DEPRECATED_API=NPY_1_7_API_VERSION)
 # generator automatically runs the preprocessor on all Fortran files
 set(CMAKE_Fortran_FLAGS_RELEASE -O3)
 
+# Uncomment when debugging a build:
+#set(CMAKE_VERBOSE_MAKEFILE ON)
+
```

Comment:
We have the VERBOSE=1 environment variable for this, is this really necessary? Debug options should not be enabled by commenting/uncommenting code. This should be set via environment flags that are passed to `pip`.